code/game/objects/items/RCD.dm

```diff
@@ -572,9 +614,22 @@ RLD
 
 /obj/item/construction/rcd/afterattack(atom/A, mob/user, proximity)
 	. = ..()
-	if(!prox_check(proximity))
-		return
-	rcd_create(A, user)
+	if (mode == RCD_CONVEYOR)	
+		if(!range_check(A, user) || !target_check(A,user)  || istype(A, /obj/machinery/conveyor) || !isopenturf(A) || istype(A, /area/shuttle))
+			to_chat(user, "<span class='warning'>Error! Invalid tile!</span>")
+			return
+		if (!linked_switch_id)
+			to_chat(user, "<span class='warning'>Error! [src] is not linked!</span>")
+			return
+		if (get_turf(A) == get_turf(user))
+			to_chat(user, "<span class='notice'>Cannot place conveyor below your feet!</span>")
+			return
```

Comment:
This seems like more of a pain than anything else. What's the point of this check xd

Reply:
Bit late on that, but I believe that's because this is how the conveyors behave when placing